### PR TITLE
Dịch "softmax-regression" - Phần 8

### LINKED ISSUE #722: Dịch "softmax-regression" - Phần 8
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_linear-networks/softmax-regression_vn.md#L381-L422

### PR DESCRIPTION
**Pull request by lyphilong**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #722 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_linear-networks/softmax-regression_vn.md [hunk 1]
```diff
@@ -394,7 +394,11 @@ The math that we used previously to define the loss $l$ still works out fine, ju
 It is the expected value of the loss for a distribution over labels.
 -->
 
-*dịch đoạn phía trên*
+Bây giờ, thay vì ta xem xét chỉ trên một kết quả thì ta hãy xem xét trên toàn bộ phân phối của tập kết quả.
+Ta có thể dùng $y$ để ký hiệu như trước đây.
+Sự khác biệt duy nhất là thay vì một vector chỉ chứa các phần tử nhị phân, giả sử là $(0, 0, 1)$, thì giờ đây ta được một vectơ xác suất chung, giả sử là $(0.1, 0.2, 0.7)$.
```
Comment (minhduc0711) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Sự khác biệt duy nhất là thay vì có một vector chỉ chứa các phần tử nhị phân, giả sử như $(0, 0, 1)$, thì giờ ta có một vector xác suất khái quát, ví dụ như $(0.1, 0.2, 0.7)$.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 2]
```diff
@@ -394,7 +394,11 @@ The math that we used previously to define the loss $l$ still works out fine, ju
...
+Trước đây, các công cụ toán học ta dùng để định nghĩa hàm loss $l$ vẫn còn khá tốt, nhưng cách giải thích chỉ dừng ở mức hơi chung chung.
```
Comment (minhduc0711) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Các công thức toán học ta dùng trước đó để định nghĩa hàm loss $l$ vẫn áp dụng tốt ở đây, chẳng qua ý tưởng của nó bây giờ khái quát hơn một chút.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 3]
```diff
@@ -394,7 +394,11 @@ The math that we used previously to define the loss $l$ still works out fine, ju
...
+Vectơ xác suất chung chính là giá trị kỳ vọng của hàm mất mát trên toàn bộ phân phối của nhãn.
```
Comment (minhduc0711) [inline chapter_linear-networks/softmax-regression_vn.md]: Chỗ này tác giả đang nói đến hàm loss
```suggestion
Nó là giá trị kỳ vọng của hàm mất mát trên phân phối của nhãn.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 4]
```diff
@@ -405,20 +409,20 @@ This loss is called the cross-entropy loss and it is one of the most commonly us
 We can demystify the name by introducing the basics of information theory.
 -->
 
-*dịch đoạn phía trên*
+Hàm mất mát này đuợc gọi là hàm mát mát cross-entropy mà nó được sử dụng rất phổ biến trong các bài toán phân loại đa lớp.
```
Comment (minhduc0711) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Hàm mất mát này đuợc gọi là hàm mát mát entropy chéo và nó là một trong những hàm mất mát phổ biến nhất dùng cho bài toán phân loại đa lớp.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 5]
```diff
@@ -405,20 +409,20 @@ This loss is called the cross-entropy loss and it is one of the most commonly us
...
+Ta có thể làm rõ tên gọi cross-entropy thông qua một vài điều cơ bản trong lý thuyết thông tin.
```
Comment (minhduc0711) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Ta có thể làm sáng tỏ cái tên entropy chéo bằng việc giới thiệu các kiến thức cơ bản trong lý thuyết thông tin.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 6]
```diff
@@ -405,20 +409,20 @@ This loss is called the cross-entropy loss and it is one of the most commonly us
...
 
 <!--
 ## Information Theory Basics
 -->
 
-## *dịch tiêu đề phía trên*
+## Lý thuyết Thông tin Cơ bản
```
Comment (minhduc0711) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
## Kiến thức Lý thuyết Thông tin cơ bản
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 7]
```diff
@@ -405,20 +409,20 @@ This loss is called the cross-entropy loss and it is one of the most commonly us
...
 
 <!--
 Information theory deals with the problem of encoding, decoding, transmitting and manipulating information (also known as data) in as concise form as possible.
 -->
 
-*dịch đoạn phía trên*
-
+Lý thuyết thông tin quan tâm đến một lớp các bài toán về mã hóa, giải mã, truyền tải và xử lý thồng tin (hay còn được gọi là dữ liệu) thành một dạng ngắn gọn nhất có thể.
```
Comment (minhduc0711) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Lý thuyết thông tin giải quyết các bài toán mã hóa, giải mã, truyền tải và xử lý thông tin (hay còn được gọi là dữ liệu) dưới dạng ngắn gọn nhất có thể.
```

**Comment by lyphilong** [comment]
Mình xin cảm ơn @minhduc0711 đã review bài của mình

File: chapter_linear-networks/softmax-regression_vn.md [hunk 8]
```diff
@@ -394,7 +394,11 @@ The math that we used previously to define the loss $l$ still works out fine, ju
 It is the expected value of the loss for a distribution over labels.
 -->
 
-*dịch đoạn phía trên*
+Bây giờ, thay vì ta xem xét chỉ trên một kết quả thì ta hãy xem xét trên toàn bộ phân phối của tập kết quả.
+Ta có thể dùng $y$ để ký hiệu như trước đây.
```
[the remark this answers is not in the document]
Comment (minhduc0711) [reply chapter_linear-networks/softmax-regression_vn.md]: @lyphilong bạn ơi mấy suggestions này có vấn đề gì không nhỉ?

[the remark this answers is not in the document]
Comment (lyphilong) [reply chapter_linear-networks/softmax-regression_vn.md]: @minhduc0711 sáng mình commit hơi gấp nên có một vài suggest mình đọc chưa kỹ nên mình chưa kịp commit á :D

[the remark this answers is not in the document]
Comment (minhduc0711) [reply chapter_linear-networks/softmax-regression_vn.md]: @lyphilong ah okay nếu bạn thấy có gì k đúng thì cứ comment nhé

[the remark this answers is not in the document]
Comment (lyphilong) [reply chapter_linear-networks/softmax-regression_vn.md]: @minhduc0711 mình mới vừa cập nhật lại á

File: chapter_linear-networks/softmax-regression_vn.md [hunk 9]
```diff
@@ -405,20 +409,20 @@ This loss is called the cross-entropy loss and it is one of the most commonly us
...
-## *dịch tiêu đề phía trên*
+## Lý thuyết Thông tin Cơ bản
 
 <!--
 Information theory deals with the problem of encoding, decoding, transmitting and manipulating information (also known as data) in as concise form as possible.
 -->
 
-*dịch đoạn phía trên*
-
+Lý thuyết thông tin giải quyết các bài toán mã hóa, giải mã, truyền tải và xử lý thông tin (hay còn được gọi là dữ liệu) dưới dạng ngắn gọn nhất có thể.
```
Comment (rootonchair) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Lý thuyết thông tin giải quyết các bài toán mã hóa, giải mã, truyền tải và xử lý thông tin (hay còn được gọi là dữ liệu) dưới dạng súc tích nhất có thể.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 10]
```diff
@@ -394,7 +394,11 @@ The math that we used previously to define the loss $l$ still works out fine, ju
 It is the expected value of the loss for a distribution over labels.
 -->
 
-*dịch đoạn phía trên*
+Giờ hãy xem xét trường hợp mà ta quan sát được toàn bộ phân phối của đầu ra thay vì chỉ một giá trị đầu ra duy nhất.
+Ta có thể biểu diễn giống hệt như trước cho $y$.
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Ta có thể biểu diễn $y$ giống hệt như trước.
```

File: chapter_linear-networks/softmax-regression_vn.md [hunk 11]
```diff
@@ -394,7 +394,11 @@ The math that we used previously to define the loss $l$ still works out fine, ju
...
+Sự khác biệt duy nhất là thay vì có một vector chỉ chứa các phần tử nhị phân, giả sử như $(0, 0, 1)$, giờ ta có một vector xác suất tổng quát, ví dụ như $(0.1, 0.2, 0.7)$.
+Các công thức toán học ta dùng trước đó để định nghĩa hàm loss $l$ vẫn áp dụng tốt ở đây, chẳng qua ý tưởng của nó bây giờ khái quát hơn một chút.
```
Comment (lkhphuc) [inline chapter_linear-networks/softmax-regression_vn.md]:
```suggestion
Các công thức toán học ta dùng trước đó để định nghĩa hàm mất mát $l$ vẫn áp dụng tốt ở đây, chẳng qua ý tưởng của nó bây giờ khái quát hơn một chút.
```
